third_party/accessibility/gfx/range/range.cc

```diff
@@ -2,18 +2,19 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "ui/gfx/range/range.h"
+#include "range.h"
 
 #include <inttypes.h>
 
 #include <algorithm>
-
-#include "base/strings/stringprintf.h"
+#include <sstream>
 
 namespace gfx {
 
 std::string Range::ToString() const {
-  return base::StringPrintf("{%" PRIu32 ",%" PRIu32 "}", start(), end());
+  std::ostringstream output;
```

Comment:
we have StringPrintf in base/string_utils.h

Reply:
This is not addressed yet

Reply:
Thanks for the reminder, fixed now